backend/src/main/java/com/google/sps/data/Match.java

```diff
@@ -13,20 +13,24 @@
   private final int duration;
   /** Time match found */
   private final long timestamp;
-  
+
   /** Constructor */
-  public Match(long id, Participant firstParticipant, Participant secondParticipant, int duration, long timestamp) {
+  public Match(
+      long id,
+      Participant firstParticipant,
+      Participant secondParticipant,
+      int duration,
+      long timestamp) {
+    // Note: Match includes entire Participants instead of just usernames to distinguish between a
+    // user participating
+    // and finding a match multiple times. This wouldn't be an issue if we deleted matches from the
+    // datastore.
```

Comment:
This looks like it would be better suited as class javadocs. Also could you elaborate in the comment on why two usernames wouldn't be sufficient?